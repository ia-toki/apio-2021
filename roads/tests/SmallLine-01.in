c10234cabdfd6eae47773fb6f163e0350fc797e1
171
0 1 766172074
1 2 931210333
2 3 725812205
3 4 639092706
4 5 802029394
5 6 634444904
6 7 666167555
7 8 825997857
8 9 851226871
9 10 843589661
10 11 970684047
11 12 748794185
12 13 791747935
13 14 948683271
14 15 978918318
15 16 829093463
16 17 648684939
17 18 812208360
18 19 723186608
19 20 854307874
20 21 847669216
21 22 836372419
22 23 805284949
23 24 937249069
24 25 654824841
25 26 517970530
26 27 500510468
27 28 957755434
28 29 617262019
29 30 633010991
30 31 705116706
31 32 837540576
32 33 654026823
33 34 642770378
34 35 568038264
35 36 501498568
36 37 621253311
37 38 637172555
38 39 976991461
39 40 977602665
40 41 904625276
41 42 583528821
42 43 752218814
43 44 842594811
44 45 864615296
45 46 725854135
46 47 896989100
47 48 950263288
48 49 883687541
49 50 671850294
50 51 900944061
51 52 964005322
52 53 894037832
53 54 897680304
54 55 901787746
55 56 641200268
56 57 958790142
57 58 984786300
58 59 640067823
59 60 615781328
60 61 923770906
61 62 959307782
62 63 894096986
63 64 565221737
64 65 503702349
65 66 717726814
66 67 954745635
67 68 822732193
68 69 993238542
69 70 938329605
70 71 960884541
71 72 612408464
72 73 876365799
73 74 520390599
74 75 627644779
75 76 671458977
76 77 951155719
77 78 713539550
78 79 863410201
79 80 501307457
80 81 572367967
81 82 568944071
82 83 841374190
83 84 760506764
84 85 567992069
85 86 579645842
86 87 985135477
87 88 912629572
88 89 779457943
89 90 574263922
90 91 713146946
91 92 549985720
92 93 537104263
93 94 606298240
94 95 998272992
95 96 685567441
96 97 917407635
97 98 717719245
98 99 951675443
99 100 785494782
100 101 542397902
101 102 895994940
102 103 756476507
103 104 624963281
104 105 991464250
105 106 959860540
106 107 656734987
107 108 640739224
108 109 769221563
109 110 633850933
110 111 501399468
111 112 505915515
112 113 954167128
113 114 778171904
114 115 715089138
115 116 573607188
116 117 685709488
117 118 967568693
118 119 626947884
119 120 756688213
120 121 769262249
121 122 720232939
122 123 756946933
123 124 967751728
124 125 971817479
125 126 758428198
126 127 880397282
127 128 506073210
128 129 544004214
129 130 955014695
130 131 880360450
131 132 808519827
132 133 560574330
133 134 669653743
134 135 821543285
135 136 520114459
136 137 905456785
137 138 727176902
138 139 612342030
139 140 689360770
140 141 990967393
141 142 724210732
142 143 862446833
143 144 995618070
144 145 677128679
145 146 633017271
146 147 514723281
147 148 856836385
148 149 531425535
149 150 815048064
150 151 537389658
151 152 651704606
152 153 870739681
153 154 527636604
154 155 631498157
155 156 779879221
156 157 523517316
157 158 983025576
158 159 608547082
159 160 739869825
160 161 915112381
161 162 935530059
162 163 949176170
163 164 606099120
164 165 714211127
165 166 704237233
166 167 741559367
167 168 950082585
168 169 818896898
169 170 783604525
