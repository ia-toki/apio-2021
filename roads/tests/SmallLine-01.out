064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
129209451152 62155967967 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
